### PR TITLE
[WL-523] Coupon Admin is able creates Discount code with 0 percentage.

### PR DESCRIPTION
**Pull request by iivic**
This PR should set the min value to 1 on benefit value input and invoice discount value input.
@marjev  @vkaracic  Please review.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @iivic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=830

File: ecommerce/static/js/views/coupon_form_view.js [hunk 1]
```diff
@@ -527,12 +529,13 @@ define([
...
                     });
-                    this.$('[name=invoice_discount_value]').attr('max', 100);
-                    this.$('[name=benefit_value]').attr('max', 100);
                     this.$('button[type=submit]').html(gettext('Create Coupon'));
                     this.$('.catalog-query').removeClass('editing');
                 }
 
+                this.$('[name=invoice_discount_value]').attr({ 'max': 100, 'min': 1});
+                this.$('[name=benefit_value]').attr({ 'max': 100, 'min': 1});
+
```
Comment (marjev) [inline ecommerce/static/js/views/coupon_form_view.js:542]: DRY:
Lines 256, 536, 537 should be replaced with a function call. The function should set specific min and max values to a specific element.

**Comment by iivic** [comment]
Comment addressed please take a quick look on the changes.

**Comment by vkaracic** [comment]
:+1:

**Comment by marjev** [comment]
👍

**Pull request merged**